src/multimeasurerest.ts

```diff
@@ -145,36 +132,19 @@ export class MultiMeasureRest extends Element {
     const padding = (right - left) * 0.1;
     left += padding;
     right -= padding;
-
-    let lineThicknessHalf;
-    if (this.#hasLineThickness) {
-      lineThicknessHalf = options.lineThickness * 0.5;
-    } else {
-      lineThicknessHalf = spacingBetweenLines * 0.25;
+    let txt = '\ue4ef'; /*restHBarLeft*/
+    const el = new Element();
+    el.setText(txt);
+    el.measureText();
+    for (let i = 1; (i + 2) * el.getWidth() + left <= right; i++) {
+      txt += '\ue4f0'; /*restHBarMiddle*/
```

Comment:
This could use some comment right before the for loop. The math is not obvious at first glance.

Reply:
done